e2e/paths/enterChildren_test.js

```diff
@@ -68,50 +68,51 @@ Scenario('completing entering child information in the c110a application', (I, e
   I.continueAndSave();
   I.seeEventSubmissionConfirmation(config.applicationActions.enterChildren);
   caseViewPage.selectTab(caseViewPage.tabs.casePeople);
-  I.seeAnswerInTab(1, 'Child 1', 'Child\'s full name', 'Timothy');
-  I.seeAnswerInTab(2, 'Child 1', 'Date of birth', '1 Aug 2015');
-  I.seeAnswerInTab(3, 'Child 1', 'Gender', 'Boy');
-  I.seeAnswerInTab(4, 'Child 1', 'Describe child\'s situation', 'Living with respondents');
-  I.seeAnswerInTab(5, 'Child 1', 'What date did they start staying here?', '1 Nov 2017');
+  I.seeAnswerInTab(1, 'Party', 'First name', 'Bran');
+  I.seeAnswerInTab(2, 'Party', 'Last name', 'Stark');
+  I.seeAnswerInTab(3, 'Party', 'Date of birth', '1 Aug 2015');
+  I.seeAnswerInTab(4, 'Party', 'Gender', 'Boy');
   I.seeAnswerInTab(1, 'Current address', 'Building and Street', 'Flat 2');
   I.seeAnswerInTab(2, 'Current address', '', 'Caversham House 15-17');
   I.seeAnswerInTab(3, 'Current address', '', 'Church Road');
   I.seeAnswerInTab(4, 'Current address', 'Town or City', 'Reading');
   I.seeAnswerInTab(5, 'Current address', 'Postcode/Zipcode', 'RG4 7AA');
   I.seeAnswerInTab(6, 'Current address', 'Country', 'United Kingdom');
-  I.seeAnswerInTab(7, 'Child 1', 'Key dates for this child', 'Tuesday the 11th');
-  I.seeAnswerInTab(8, 'Child 1', 'Brief summary of care and contact plan', 'care plan summary');
-  I.seeAnswerInTab(9, 'Child 1', 'Are you considering adoption at this stage?', 'No');
-  I.seeAnswerInTab(10, 'Child 1', 'Mother\'s full name', 'Laura Smith');
-  I.seeAnswerInTab(11, 'Child 1', 'Father\'s full name', 'David Smith');
-  I.seeAnswerInTab(12, 'Child 1', 'Does the father have parental responsibility?', 'Yes');
-  I.seeAnswerInTab(13, 'Child 1', 'Name of social worker', 'James Jackson');
-  I.seeAnswerInTab(14, 'Child 1', 'Social worker\'s telephone number', '01234567');
-  I.seeAnswerInTab(15, 'Child 1', 'Does the child have any additional needs?', 'No');
-  I.seeAnswerInTab(16, 'Child 1', 'Do you need contact details hidden from other parties?', 'No');
-  I.seeAnswerInTab(17, 'Child 1', 'Do you believe this child will have problems with litigation capacity (understanding what\'s happening in the case)', 'Yes');
-  I.seeAnswerInTab(18, 'Child 1', 'Give details, including assessment outcomes and referrals to health services', 'mock reason');
+  I.seeAnswerInTab(6, 'Party', 'Describe child\'s situation', 'Living with respondents');
```

Comment:
The ordering of these fields should not have changed if possible. Child's situation used to be above address.